src/libs/actions/Report.ts

```diff
@@ -2237,6 +2237,9 @@ function clearPolicyRoomNameErrors(reportID: string) {
         pendingFields: {
             reportName: null,
         },
+        errors: {
```

Comment:
@nodebrute Thanks for the PR. While the current changes will solve this specific problem, I notice that this will lead into another issue as demonstrated in the video below.  Here reload of the report using `openReport` with cause the `errorFields` to be reset thereby leaving the LHN with GBR which is not correct. I think instead of removing the `errors` here, we should not add `errors` [here](https://github.com/Expensify/App/blob/2026f62b5074d892163f837ca7b030b84c8e350f/src/libs/actions/Report.ts#L656C2-L658C19). Would this work?

https://github.com/Expensify/App/assets/3004659/c088a820-55d3-4ecb-97eb-8416c50d560e

Reply:
> However, I would suggest that we add `errorFields` to the failure data.

@Nodebrute What problem would it cause if we do not add `errorFields` to the failure data?

Reply:
@Nodebrute Oh! Now the error itself does not display for me. Can you please test this?

Reply:
I think resetting of `errorFields` in BE for `openReport` may be a problem. Let us check with internal engineer.

Reply:
@danieldoglas The BE is resetting the `errorFields` for `Invalid group chat name` when `openReport` is invoked for the report. Is this intentional? This is causing the issue as demonstrated in the video [here](https://github.com/Expensify/App/pull/44130#discussion_r1650369474). 

![Screenshot 2024-06-27 at 9 56 49 AM](https://github.com/Expensify/App/assets/3004659/05e9ffbd-78e2-4218-8a64-20cb556a69fe)

Reply:
@danieldoglas Oh! Sorry, I think I should have been more clear here. 

Here, when we provide an invalid group name i.e. `adoadoadoajdoajdoadoajdoajadoadoadoajdoajdoadoajdoajadoadoadoajdoajdoadoajdoajadoadoadoajdoajdoadoaỏ`, `UpdateGroupChatName` API returns `errorFields` with `Invalid group chat name` as shown below. 

However, when the user navigates away and comes back to the same group report via LHN, `openReport` (shared screenshot below) resets the `errorFields` 

This is causing an issue where error is displayed for some time in the group chat report but disappears when `openReport` gets called. 

I think `openReport` resetting the errorFields in BE does not seem to be intentional. 
What do you think?

![Screenshot 2024-06-29 at 11 26 59 AM](https://github.com/Expensify/App/assets/3004659/400bb7e8-1572-48cc-9b44-b07b797009fb)

![Screenshot 2024-06-29 at 11 30 32 AM](https://github.com/Expensify/App/assets/3004659/07b6895e-c48c-4c9f-9b41-7a2431f26ed8)

Reply:
Sorry,  missed this message.

> I think openReport resetting the errorFields in BE does not seem to be intentional.

Interesting, but the part that is more curious to me is that  it is returning errorFields as an object and only the `notFound` property. Considering that the error was `reportName`, it shouldn't be resetting the `reportName` error on a merge operation in onyx.

Reply:
But then, at the same time, `Openreport` will return the correct name of the report, so it wouldn't really show the name it failed to set anymore, right?

Reply:
> But then, at the same time, `Openreport` will return the correct name of the report, so it wouldn't really show the name it failed to set anymore, right?

That's correct. I think setting `errorFields` with error is a way to inform the user that it failed to set the name.

Reply:
> Interesting, but the part that is more curious to me is that it is returning errorFields as an object and only the `notFound` property. Considering that the error was `reportName`, it shouldn't be resetting the `reportName` error on a merge operation in onyx.

@danieldoglas That's very interesting. Didn't quite think from the perspective of `Notfound` property though. In that case, not sending `errorFields` itself may solve the problem. Anyway, `Notfound` does not seem intentional. What do you think?

Reply:
Sorry:

- errorFields is an object, that may contain several types of errors
- For some reason, when we send it with `notFound`, it is deleting the `reportName` error

Do you know why that's happening? Are we returning a method SET instead of MERGE when returning the policy?

Reply:
> * errorFields is an object, that may contain several types of errors
> * For some reason, when we send it with `notFound`, it is deleting the `reportName` error
> 
> Do you know why that's happening? Are we returning a method SET instead of MERGE when returning the policy?

Ah! I get your point. @Nodebrute Can you please investigate this?